pkg/volume/local/local.go

```diff
@@ -204,8 +210,18 @@ func (m *localVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 		glog.Errorf("cannot validate mount point: %s %v", dir, err)
 		return err
 	}
-	if !notMnt {
```

Comment:
@sbezverk - Thanks for the explanation. Since, this object isn't used here anymore. It could be removed. It could be initialized later, where this particular name was used.